x-pack/plugins/security_solution/public/flyout/document_details/right/components/insights_section.tsx

```diff
@@ -28,8 +33,11 @@ export const InsightsSection = memo(() => {
   const { getFieldsData } = useRightPanelContext();
   const eventKind = getField(getFieldsData('event.kind'));
 
-  const expanded = useExpandSection({ title: KEY, defaultValue: false });
-
+  const { activeStep, isTourShown } = useTourContext();
+  const isGuidedOnboardingTourShown =
+    isTourShown(SecurityStepId.alertsCases) && activeStep === AlertsCasesTourSteps.viewCase;
+  const expanded =
```

Comment:
Expand the insight section automatically after the test case created, so the relevant cases can be fetched.